code/__DEFINES/subsystems.dm

```diff
@@ -58,8 +58,8 @@
 #define INIT_ORDER_JOBS 12
 #define INIT_ORDER_TICKER 11
 #define INIT_ORDER_MAPPING 10
-#define INIT_ORDER_ATOMS 9
-#define INIT_ORDER_NETWORKS 8
+#define INIT_ORDER_NETWORKS 9
+#define INIT_ORDER_ATOMS 8
```

Comment:
Keep the numbered ordering

Reply:
I do. Ordering is the same, first 9 then 8.

Reply:
bline nevermind